### PR TITLE
wiki(meandeck): add alternative piles chapter

### PR DESCRIPTION
**Pull request by Doishy**

### CODE REVIEW

**Comment by Doishy** [comment]
Drafted up. Ready for review. Would like especially @angrybacon to check DRC piles and @d8dk32 to check Echo ones. 
Happy if you want to add additional examples of either.

File: markdown/chapters/meandeck/alternative-piles.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+---
+authors: Doishy
+title: Alternative Pile Building
+---
+
+## Contents
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:6]:
```suggestion
## Table of Contents
```

The `remark` plugin looks for `Table of Content` specifically 🤓

File: markdown/chapters/meandeck/alternative-piles.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+- SI
+- Em
+- Island
+- CoS
+- TO
+:::
+
+In theory you can replace Emrakul with any card (including lands) however
+the uncounterable nature, the take an extra turn on casting (which still 
+works with Shelldock!) and the attack for 15 with Annihilator 6 all make it
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:36]:
```suggestion
works with Shelldock!) and the attack for 15 with *Annihilator 6* all make it
```

File: markdown/chapters/meandeck/alternative-piles.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+probably the most effective tool to use. 
+
+## Piles using Dragon's Rage Channeler
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:39]:
```suggestion
## Piles Using Dragon's Rage Channeler
```

File: markdown/chapters/meandeck/alternative-piles.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+Some people have tried various creature shells alongside the Doomsday package
+and one of the more interesting ones for use as part of the Doomsday
+resolution is card:[Dragon's Rage Channeler] (DRC). Each spell cast can let you "dig"
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:43]:
```suggestion
resolution is card:[Dragon's Rage Channeler] (DRC). Each spell cast can let you *dig*
```

Also, feel free to add an entry here `src/tools/scryfall/cards.ts` as the card makes sense both for and against Doomsday as an archetype (and it's a pain to type, I often forget where the `'s` actually is)

File: markdown/chapters/meandeck/alternative-piles.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+one additional card deeper through your pile thanks to the Survail ability.
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:44]:
```suggestion
one additional card deeper through your pile thanks to the *Surveil* ability.
```

File: markdown/chapters/meandeck/alternative-piles.md [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+:::row{variant=pile}
+- DA
+- LED
+- SW
+- LP
+- TO
+:::
+
+The most basic example above allows you to turn any regular cantrip into a 
+pseudo copy of card:[Con] by giving it a Surveil 1 effect attached. This allows
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:58]:
```suggestion
pseudo copy of card:[Con] by giving it a *Surveil 1* effect attached. This allows
```

File: markdown/chapters/meandeck/alternative-piles.md [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+- SW
+- BS
+- LED
+- EoA
+- TO
+:::
+
+Here you can execute a Brainstorm pile even if you are at a low life total without the
+use of card:[SW] or if you don't have access to tools like card:[IU] or card:[Con] which
+still allows you to get to 0 cards left in the deck. You can follow the listed steps:
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:74]:
```suggestion
still allows you to get to 0 card left in the deck. You can follow the listed steps:
```

File: markdown/chapters/meandeck/alternative-piles.md [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+1. Cast BS, do not Surveil and put back TO and X
+1. Cast LED, Surveil X into the graveyard
+1. Cycle Edge, holding priority and cracking LED for {U}{U}{U} draw Oracle
+1. Cast Thassa's Oracle (TO) with an empty library
+
+## Double Oracle Piles
+
+Often you will see people running 2 card:[TO] in the maindeck or 1 main and 1 in the side.
+This card naturally fits into many pre-existing piles which we have covered already and give
+the option to "try and try again" when looking to beat cards like card:[Endurance].
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:86]:
```suggestion
the option to *try and try again* when looking to beat cards like card:[Endurance] and :card[Memory's Journey].
```

File: markdown/chapters/meandeck/alternative-piles.md [hunk 9]
```diff
@@ -0,0 +1,203 @@
...
+:::row{variant=pile}
+- IU
+- LP
+- TO
+- CoS
+- TO
+:::
+
+This is a very basic take on the standard card:[IU] pile which is often used. It can be 
+effective in "risking" jamming a pile without putting too many cards into the graveyard
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:99]:
```suggestion
effective in *risking* jamming a pile without putting too many cards into the graveyard
```

File: markdown/chapters/meandeck/alternative-piles.md [hunk 10]
```diff
@@ -0,0 +1,203 @@
...
+which may be returned with Endurance and also still being able to have another Oracle
+in reserve if the first one doesn't work out. What is nice is that even if they were to
+Endurance you here you library would become (randomised after the first TO) some variation
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:102]:
```suggestion
Endurance you here, your library would become (randomised past the first TO, still on top of the library) some variation
```

File: markdown/chapters/meandeck/alternative-piles.md [hunk 11]
```diff
@@ -0,0 +1,203 @@
...
+of the following:
+
+:::row{variant=pile}
+- TO
+- DD
+- LP
+- IU
+:::
+
+As a result if they do cast and Endurance then you can pass and simply draw the second Oracle
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:112]:
```suggestion
As a result if they do cast an Endurance then you can pass and simply draw the second Oracle
```

File: markdown/chapters/meandeck/alternative-piles.md [hunk 12]
```diff
@@ -0,0 +1,203 @@
...
+- EoA
+- TO
+:::
+
+Here you might have a scenario where you want to try and win quickly but suspect they may have a Red
+Blast effect. In which case you can pass, allow Ragavan to exile SW, and try to jam the first
+Oracle off of Cavern (assuming the cantrip resolves of course). If that Oracle is dealt with then
+you can pass again, Ragavan exiles the card:[EoA] and you try casting the second Oracle.
+
+## Slow Piles using Interactive Elements
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:132]:
```suggestion
## Slow Piles Using Interactive Elements
```

File: markdown/chapters/meandeck/alternative-piles.md [hunk 13]
```diff
@@ -0,0 +1,203 @@
...
+
+:::row{variant=pile}
+- Baleful Strix
+- EoA
+- Baleful Strix
+- CoS
+- TO
+:::
+
+You can also combine elements to use Teferi to bounce Strix for additional card draw.
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:164]:
```suggestion
You can also combine elements to use Teferi to bounce Strix for additional card draw and still maintain a higher {U} Devotion.
```

File: markdown/chapters/meandeck/alternative-piles.md [hunk 14]
```diff
@@ -0,0 +1,203 @@
...
+
+:::row{variant=pile}
+- LED
+- PoN
+- TO
+- LP
+- LP
+:::
+
+There is a much more extensive look at Echo of Eons in the DDFT section of the wiki.
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:197]:
```suggestion
There is a much more extensive look at Echo of Eons in the [DDFT section][ddft:echo] of the wiki.

[ddft:echo]: /ddft/echo
```

File: markdown/chapters/meandeck/alternative-piles.md [hunk 15]
```diff
@@ -0,0 +1,203 @@
...
+
+## Next Steps
+
+You don't need to have anything here memorised but do use the information presented
+here as a way of looking at alternative ways perhaps you can build piles with either
```
Comment (angrybacon) [inline markdown/chapters/meandeck/alternative-piles.md:202]:
```suggestion
here as a way of looking at alternative ways. Perhaps you can build piles with either
```

**Review by angrybacon — CHANGES_REQUESTED** [review]
Review of the existing done, I'll fill paragraphs later so don't bother with that yet. If anything relevant comes to mind I'll add it in the DRC sections

File: markdown/chapters/meandeck/alternative-piles.md (hunk 7 above)
Comment (Doishy) [reply markdown/chapters/meandeck/alternative-piles.md:74]: Not sure why this change suggested, should be "cards"

**Review by angrybacon — DISMISSED** [review]

**Comment by d8dk32** [comment]
@Doishy I would maybe add a little to your intro to the Echo section addressing how its specifically useful against Endurance, since I think that's the main draw of Echo in Meandeck.

**Comment by Doishy** [comment]
Note to self, include either here or another PR to add "piles that play around draw sevens"

**Review by angrybacon — APPROVED** [review]

**Pull request merged**